editor.planx.uk/src/@planx/graph/index.ts

```diff
@@ -209,6 +220,8 @@ export const clone =
       else if (!draft[toParent]) throw new Error("toParent not found");
       else if (draft[toParent].edges?.includes(id))
         throw new Error("cannot clone to same parent");
+      else if (isSectionNodeType(id, graph))
+        throw new Error("cannot clone sections");
```

Comment:
cloning is a bit of an interesting case here because as long as our `_add()` & `move()` rules for sections are working as expected, then cloning would be implicitly prohibited because you can never clone to the same parent (eg always `_root` in the case of a valid section node) - but feels good to be explicit about intended behavior here to me